04_03_defuse_0_6_2_Dockerfile

```diff
@@ -0,0 +1,29 @@
+# ProTECT Dockerfile for deFuse 0.6.2
+#
+# Build with: sudo docker build --force-rm --no-cache -t aarjunrao/defuse:0.6.2 - < 04_03_defuse_4_3_Dockerfile
+# Run with sudo docker run -v <IO_folder>:/data aarjunrao/defuse:0.6.2 <parameters>
+
+# Use pimmuno
+FROM aarjunrao/pimmuno:pimmuno
+
+MAINTAINER Jacob Pfeil, jpfeil@ucsc.edu
+
+RUN echo 'deb http://us.archive.ubuntu.com/ubuntu trusty main multiverse' >> /etc/apt/sources.list
+
+RUN apt-get update && apt-get install -y libboost-all-dev \
+                                         samtools \
+                                         bowtie \
```

Comment:
I'm worried that we're not enforcing any version restrictions on these dependencies. I think we should. Let's talk to Hannes and John